lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -23,8 +24,10 @@ const MINIMAL_LEGIBLE_FONT_SIZE_PX = 12;
 const MAX_NODES_VISITED = 500;
 const MAX_NODES_ANALYZED = 50;
 
+const Driver = require('../../driver.js'); // eslint-disable-line no-unused-vars
+
 /**
- * @param {!Node} node
+ * @param {LH.Artifacts.FontSize.DomNodeWithParent} node
```

Comment:
should we have any guidance on when to break out into separate interface vs. do it inline and the `LH.Artifacts['Artifact']` method?

Reply:
I'm open to whatever. We could always access by property (even if there's a distinct interface) just to keep it consistent. Or we could always define a distinct interface, even if the type is `string|null` (really the only downside is excessive file size, but `d.ts` files always end up huge).

The rule I was following was strictly "can it fit easily on one line" and had no other significant thought put into it :)

Reply:
gotcha, wfm!